docs/managed-tests/security.md

```diff
@@ -0,0 +1,50 @@
+# Security Tests
```

Comment:
Same comment on this one as the plugin check; I think the original was better.

Reply:
Agree 100%, that was an oversight. Let me know what you think of this version, and similar to the Plugin Check, feel free to modify it in any way shape or form.

![image](https://github.com/user-attachments/assets/4481fcc2-3bf0-456f-958d-031818c9eed8)